### DISCUSSION TITLE
Popdialog Component

### DISCUSSION THREAD

**Discussion by kaptinlin** [mantinedev/mantine#2809] [Feature requests]
https://arco.design/react/en-US/components/popconfirm
https://semi.design/en-US/feedback/popconfirm
![image](https://user-images.githubusercontent.com/999331/198077288-c56c0481-fd37-4559-9b25-41af1ddc2a1d.png)

**Comment by rtivital** [comment]
We already provide Popover and HoverCards components, they do the same thing
